go.mod

```diff
@@ -4,7 +4,7 @@ go 1.21
 
 require (
 	github.com/NearNodeFlash/lustre-fs-operator v0.0.1-0.20240709160956-40dcbac0aadf
-	github.com/NearNodeFlash/nnf-sos v0.0.1-0.20240816013414-a2246392cd0a
+	github.com/NearNodeFlash/nnf-sos v0.0.1-0.20240819183331-7ebf170c8f2c
```

Comment:
I think nnf-sos will be updated after this version. Could you pull the latest before you merge this PR?

Reply:
Yes, I need to update this once the sos PR is merged.

Reply:
Done